app/views/candidate_interface/personal_details/nationalities/_shared_form.html.erb

```diff
@@ -22,9 +22,27 @@
     multiple: true,
     label: { text: 'Citizen of a different country' },
   ) do %>
-    <%= f.govuk_collection_select :other_nationality1, select_nationality_options, :id, :name, label: { text: t('application_form.personal_details.nationality.label') } %>
-    <%= f.govuk_collection_select :other_nationality2, select_nationality_options, :id, :name, label: { text: t('application_form.personal_details.nationality.label') } %>
-    <%= f.govuk_collection_select :other_nationality3, select_nationality_options, :id, :name, label: { text: t('application_form.personal_details.nationality.label') } %>
+    <%= f.govuk_collection_select(
+      :other_nationality1,
+      select_nationality_options,
+      :id,
+      :name,
+      label: ->{ safe_join([tag.span('First ', class: 'govuk-visually-hidden'), t('application_form.personal_details.nationality.label')]) },
```

Comment:
This seems like quite a fundamental change, but I don't really a sense from the commit message why this has occurred. A more descriptive commit in this instance might serve as useful documentation further down the line?